mathics/builtin/exptrig.py

```diff
@@ -3,7 +3,6 @@
 
 """
 Exponential, trigonometric and hyperbolic functions
-
```

Comment:
Looks like a bunch of whitespace changes got applied by your editor. Can you fix that up please?